at_sync_ui_flutter/example/ios/Podfile.lock

```diff
@@ -39,17 +41,21 @@ PODS:
   - file_picker (0.0.1):
     - DKImagePickerController/PhotoGallery
     - Flutter
-  - file_saver (0.0.1):
-    - Flutter
   - Flutter (1.0.0)
   - flutter_keychain (0.0.1):
     - Flutter
   - flutter_qr_reader (0.0.1):
```

Comment:
@yahu1031 at_sync_ui is using older version of at_onboarding_flutter.
that's why it still has `flutter_qr_reader` as dependency.
This will be removed once we deploy new version of at_onboarding_flutter.